experiment/template/eVOLVER.py

```diff
@@ -126,6 +135,13 @@ def on_activecalibrations(self, data):
                                 self._create_file(x, param + '_raw', defaults=[exp_str])
                     break
 
+    def on_serialexception(self, data):
+        logger.error('Failed command!')
+        logger.error(data)
+        if data in self.commands and retry_count <= RETRY_LIMIT:
+            self.emit('command', data, namespace = '/dpu-evolver')
+            retry_count = retry_count + 1
```

Comment:
Should be `self.retry_count += 1`?

Reply:
I've been switching between languages too much lately, can't remember how to increment variables anymore :joy: